index.html

```diff
@@ -0,0 +1,130 @@
+    <link rel="stylesheet" type="text/css" media="screen and (min-width: 747px)" href="./css/desktop.css">
+</head>
+<body>
+    <header>
+        <nav class="dk top-nav-container">
+            <article class="top-nav">
+                <ul class="social-icon">
+                    <li><img src="./assets/social-icon/facebook_f_64px.png" alt="facebook"></li>
+                    <li><img src="./assets/social-icon/twitter_64px.png" alt="twitter"></li>
+                </ul>
+                <ul class="h-link">
+                    <li><a href="#">English</a></li>
+                    <li><a href="#">My Page</a></li>
+                    <li><a href="#">Logout</a></li>
+                </ul>
+            </article>
+        </nav>
+        <nav class="main-nav">
+            <section class="mobile-menu mb">
+                <label class="hamburger-menu mb" for="menu-checkbox">
+                    <input type="checkbox" id="menu-checkbox" />
+                </label>
+                <aside class="sidebar">
+                    <a class="new-stuff" href="#">Logout</a>
+                    <a id="about" href="./pages/about.html">About</a>
+                    <a id="program" href="#">Program</a>
+                    <a id="join" href="#">Join</a>
+                    <a id= "sponsor" href="#">Sponsor</a>
+                    <a id="news" href="#">News</a>
+                    <a class="CC" href="#">CC Campaign</a>
+                </aside>
+            </section>
+            <a href="#" class="logo dk">BLK-NG</a>
+            <article class="main-nav-link dk">
+                <a id="about" href="./pages/about.html">About</a>
+                <a id="program" href="#">Program</a>
+                <a id="join" href="#">Join</a>
+                <a id= "sponsor" href="#">Sponsor</a>
+                <a id="news" href="#">News</a>
+                <a class="CC" href="#">CC Campaign</a>
+            </article>
+        </nav>
+        <section class="headline-container">
+            <article class="headline">
+                <h2>"Hello! Sharing world"</h2>
+                <h2>CREATIVE COMMONS<br>GLOBAL SUBMMIT<br>2023</h2>
+                <p class="intro-txt">
+                    A joyfull celebration believing in the value of openness and sharing creating 
+                    a positive change with people from over 80 countries is atking palce in Octobar, in Lagos
+                </p>
+                <h2 class="time">2023.10.15(THU) ~ 16(FRI)</h2>
+                <p>@ National Museum of Nigeria, Art Center Lagos and more</p>
+            </article>
+        </section>
+    </header>
+    <section id="main-program">
+        <h2 class="program-title wht">Main Program</h2>
+        <section id="cards">
+            <article class="card">
+                <img src="./assets/icons/home_iot_device_FILL0_wght400_GRAD0_opsz48.svg" alt="home_iot_device">
+                <h2>Home IoT</h2>
```

Comment:
- [ ] 👏 Great job on using the h2 tag for the main title in this section! However, for the subheadings or nested headings as in the card, it's important to use h3 to h6 tags for better semantic structure. To improve the semantic structure of your code, I recommend updating the subheadings to use appropriate heading tags such as h3, h4, h5, or h6 based on their hierarchical importance. To learn more check [this documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#accessibility_concerns).

By following proper semantic HTML structure, your webpage will be more accessible and better optimized for search engines. Keep up the good work, and happy coding! 🌟💻🚀